pxr/usd/usdLux/overview.dox

```diff
@@ -101,11 +101,54 @@ constraints, such as to attach lights to moving geometry.  This is
 consistent with USD's policy of storing the computed result of rigging,
 not the rigging itself.
 
+\subsection usdLux_quantities Quantities and Units
+
+Most renderers consuming OpenUSD today are RGB renderers, rather than spectral.
+Units in RGB renderers are tricky to define as each of the red, green and blue
+channels transported by the renderer represents the convolution of a spectral
+exposure distribution, e.g. CIE Illuminant D65, with a sensor response function, e.g.
+CIE 1931 𝓍̅. Thus the main quantity in an RGB renderer is neither radiance nor
+luminance, but "integrated radiance" or "tristimulus weight".
+
+The emision of a default light with `intensity` 1 and `color` [1, 1, 1] is an
+Illuminant D spectral power distribution with chromaticity matching the rendering
+colour space white point, normalized such that a ray normally incident upon the
+sensor with EV0 exposure settings will generate a pixel value of [1, 1, 1] in the
+rendering colour space.
+
+Given the above definition, that means that the luminance of said default light
+will be 1 *nit* and its emission spectral radiance distribution is easily computed by appropriate
+normalization.
+
+Note that some colour spaces, most notably ACES, define their white points by
+chromaticity coordinates that do not exactly line up to any value of a standard illuminant.
+In these cases, a spectral renderer should choose the closest Illuminant D spectrum
+for the lights' emission (the *rendering illuminant)* and perform chromatic
+adaptation to transform the rendered image to the rendering colour space.
+The method of "uplifting" an RGB colour to a spectral distribution is unspecified
+other than that it should round-trip under the rendering illuminant to the limits
+of numerical accuracy.
+
+\subsubsection usdLux_quantities_exposure Exposure
+
+In order to generate an image, a camera effectively integrates irradiance at a patch
+on the sensor over time to get exposure, which is converted to an electrical signal.
+The default behaviour of many renderers is to ignore this, which as we have seen means
+that 1 nit incident on the sensor results in an output signal of 1.0.
+
+1 nit is a tiny amount of light, equivalent by definition to a single candle viewed at a distance. Thus
+using physically plausible values for light brightnesses will result in blown out images
+unless exposure is taken into account. A full physical camera model would be a good
+future addition to OpenUSD, but there is already a means of communicating sensor
+exposure via `UsdGeomCamera::GetExposureAttr()` and it is expected that all renderers
+implementing UsdLux must respect the value of that attribute by scaling the rendered
+image by `pow(2, exposure)`.
+
 \subsection usdLux_Behavior Properties & Behavior
 
 Colors specified in attributes are in energy-linear terms,
```

Comment:
This section is now obsolete. The colorspace metadata must conform to the available colorimetry defined by GfColor/ColorSpace. Further the UsdStage::SetColorConfiguration() **does not** have any bearing on rendering. Within the rendering domain, colorimetry is strictly reduced to that specified by ocio/nanocolor as implemented by gfColor*. The mention of SetColorConfiguration should be removed in this context, as it is used to set color configurations governed by OCIO, such as the viewing transform. Color configurations, explicitly can not influence the colorimetry defined normatively by the ocio/nanocolor.

Reply:
Are you saying that the attribute colorSpace metadata should apply, but the stage color configuration should not?  If so, what should the default colorSpace be when an attribute doesn't explicitly specify?

Also - this implies that we're now creating two classes of color attributes - those to which the stage level SetColorConfiguration applies, and those to which it doesn't.  I feel that there should be some programmatic way to query this, rather than relying on an end user knowing which attributes it applies to and those to which it doesn't. 
 (Though I don't necessarily think we should gate these changes until the presence of such an API - just that they should be on the near-term roadmap.)
 
 Also - just wanted to make clear that I'm not trying to argue against this approach, just pointing out some resulting considerations, and trying to get additional clarification!

Reply:
UsdStage::SetColorConfiguration doesn't apply to the rendering color space at all, it refers to the viewing transform that is applied to the final framebuffer. 

So we're not creating two classes of color attributes at all.

As to the default color space to be used for scene referred colors, that's a topic for a new color space schema, not yet published. In the mean time, the default is what it is today, which is linear with the primaries and white point of rec bt.709.